docs/release/release-tasks.md

```diff
@@ -93,26 +93,25 @@ is used for e.g. local development and e2e tests. We also modify tests so that t
 
 This comes down to changing occurrences of the old version to the new version, e.g. `v1.3` to `v1.4`:
```

Comment:
Some more:
* https://github.com/kubernetes-sigs/cluster-api/blob/187e4163a2eeabe930dda1e89f61a3651db9f0d9/test/e2e/data/infrastructure-docker/README.md#L12 (should be updated or written more generically)
* [Clusterctl upgrade test coverage](https://main.cluster-api.sigs.k8s.io/clusterctl/commands/upgrade.html#clusterctl-upgrade-test-coverage)

Reply:
I think otherwise it's fine